src/KeyUnit.cpp

```diff
@@ -90,6 +90,16 @@ void KeyUnit::compileAll()
     }
 }
 
+TKey * KeyUnit::findKey( QString & name ) {
```

Comment:
As this method just returns the _first_ `TKey` pointer in the `QMap` which has the given reference `name` as the key there is no need to manually iterate through it - this can be entirely replaced with:

    TKey * KeyUnit::findKey( QString & name )
    {
        return mLookupTable.value( name );
    }

which is a bit simpler...! :nerd_face: